platformio.ini

```diff
@@ -871,3 +872,22 @@ build_flags =
   ${com-arduino-low-memory.build_flags}
   '-DGateway_Name="OMG_1_FL"'
   '-DZactuatorFASTLED="FASTLED"'
+
+[env:sonoff-rfbridge-direct]
+platform = ${com.esp8266_platform}
+board = esp8285
+lib_deps = 
+	${com-esp.lib_deps}
+	${libraries.esppilight}
+	puuu/ESPiLight@^0.17.0
```

Comment:
You are right mdns is not necessary here.
You should also remove : 
```
puuu/ESPiLight@^0.17.0
```
Since it is called by 
```
${libraries.esppilight}
```